Utils/Dataflow/pyDKB/dataflow/stage/tests/test_ProcessorStage.py

```diff
@@ -170,6 +170,24 @@ def f(self):
         setattr(ProcessorStageArgsTestCase, 'test_%s_%s' % (arg, val), f)
 
 
+def add_arg_incorrect(arg, short=False):
+    if short:
+        val = 'i'
```

Comment:
Maybe it would be better to use here `'incorrect`' as value instead of `'i'`? 
I have mentioned it earlier, but the [comment](https://github.com/PanDAWMS/dkb/commit/a5e428b8f4ffcc4e28964f48a29b5e31ecdb9cbf#r32009968) was buried after `push --force` :) -- values for the parameters being tested are more likely to be one-char length, so using the longer word we are almost guaranteed from hitting a new valid value by chance.

Reply:
Done.